### PR TITLE
[TIMOB-17361] Android: Initial CardView support

### PR DESCRIPTION
**Pull request by ashcoding**
Jira: https://jira.appcelerator.org/browse/TIMOB-17361

### CODE REVIEW

**Comment by ashcoding** [comment]
App.js test code:

**Note:** A png 'square.png' is used in this test code.
```
var win = Ti.UI.createWindow({
  backgroundColor: 'white',
  layout: 'vertical',
  title: 'Card View Demo'
});

///////////////////////////

var win1 = Ti.UI.createWindow({
  backgroundColor:'white',
  title: 'Card Demo 1'
});

function createRow(i) {
	var card1 = Ti.UI.Android.createCardView({
	  contentPadding: 20,
	  cardCornerRadius: 10,
	  cardUseCompatPadding: true
	  
	});
	
	var internalView = Titanium.UI.createView({
	});	
	
	var label1 = Ti.UI.createLabel({	
	  top:5,
	  color:'black',
	  text: 'Below this is a purple square',
	});
	internalView.add(label1);
	
	var image = Ti.UI.createImageView({
	  top:50,
	  image:'square.png'
	});
	internalView.add(image);
	
	card1.add(internalView);
	
	return card1;
}

var scrollView = Ti.UI.createScrollView({
  layout: 'vertical'
});

for(var i = 0; i <= 20; i++){
var row = createRow(i);
  scrollView.add(row);
}
win1.add(scrollView);

/////////////////////////////

var win2 = Ti.UI.createWindow({
  backgroundColor: 'white',
  layout: 'vertical',
  title: 'Card Demo 2'
});

var card1 = Ti.UI.Android.createCardView({
  top: 30,
  contentPadding: 50,
  cardElevation: 40,
  cardBackgroundColor: 'yellow',
  cardCornerRadius: 120,
  width: 250, height: 200
});

var label1 = Ti.UI.createLabel({

  color:'blue',
  text: 'A long label with\na few line breaks\nand unicode (UTF8)\nsymbols such as\na white chess piece \u2655\nand the euro symbol \u20ac\nlooks like this!\n',
  textAlign: Ti.UI.TEXT_ALIGNMENT_LEFT,

});

card1.add(label1);

var card2 = Ti.UI.Android.createCardView({
  top: -80,
  contentPadding: 200,  
  cardBackgroundColor: 'red',
  cardCornerRadius: 20,
  width: 300, height: 200
});


var image = Ti.UI.createImageView({
  image:'square.png'
});

card2.add(image);

var card3 = Ti.UI.Android.createCardView({
  top: -80,
  width: 100, height: 100
});

var label2 = Ti.UI.createLabel({

  color:'black',
  text: 'Let there be text',
  textAlign: Ti.UI.TEXT_ALIGNMENT_LEFT,

});

card3.add(label2);

win2.add(card1);
win2.add(card2);
win2.add(card3);

/////////////////

var button1 = Titanium.UI.createButton({
   title: 'Demo 1',
   top: 10,
   width: 100,
   height: 50
});
button1.addEventListener('click',function(e)
{
  win1.open();
});


var button2 = Titanium.UI.createButton({
   title: 'Demo 2',
   top: 10,
   width: 100,
   height: 50
});
button2.addEventListener('click',function(e)
{
  win2.open();
});

win.add(button1);
win.add(button2);
win.open();
```

File: android/modules/cardview/R.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+int attr cardBackgroundColor 0x7f010000
```
Comment (hieupham007) [inline android/modules/cardview/R.txt]: why do we need this? It's a generated file.

**Comment by ashcoding** [comment]
Removed `R.txt`

**Comment by hieupham007** [comment]
Code reviewed and functionally tested. Request **accepted**

**Comment by ashcoding** [comment]
Yup. This is ready to be merged if everything is okay.

**Pull request merged**